perf/index.js

```diff
@@ -0,0 +1,61 @@
+const levenshteinEditDistance = require('levenshtein-edit-distance');
+const fastLevenshtein = require('fast-levenshtein');
+const jsLevenshtein = require('js-levenshtein');
+const talisman = require('talisman/metrics/distance/levenshtein');
+const leven = require('leven');
+const jsLevenshteinEsm = require('../');
+const paragraphs = require('./data/paragraphs');
+const sentences = require('./data/sentences');
+const words = require('./data/words');
+
+if (!Object.entries) {
```

Comment:
shouldn't this always exist? (we run the perf tests in node)
if its because we're trying to support an old node, i'd rather we just drop that version to avoid clutter. what do you think?